include/grpc/impl/codegen/port_platform.h

```diff
@@ -196,7 +196,7 @@
 #define GPR_GCC_TLS 1
 #define GPR_LINUX 1
 #define GPR_LINUX_LOG
-#define GPR_LINUX_MULTIPOLL_WITH_EPOLL 1
+#define GPR_POSIX_MULTIPOLL_WITH_POLL 1
```

Comment:
I intend to remove this before submission.  I wanted to run all the linux c/wrapped language tests using this shim to be more confident that it works.  

Eventually I want to add a whole NaCl build column to Jenkins and run the c tests using a nacl build, but that is outside the scope of this PR.

This was a good review catch btw :)